app/containers/App/selectors.js

```diff
@@ -111,7 +116,20 @@ const makeEntitySelector = () => createSelector(
 
 const makeEntityMapSelector = () => createSelector(
```

Comment:
@tim-field attempting to join in the username from users tables but it feels hacky. also I am not sure if we wanted to do ythings like this here in the global selectors or in the containers (eg ActionEdit)....